main/res/drawable/ic_menu_pocket_query.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you use vector drawables even for older platforms (8+)? Are they handled by AppCompat or generated as png by the build in such a case?